### ISSUE TITLE
Issue with reading a mail's from email address in outlook add-in

### ISSUE THREAD

**Issue by swasti1994** [OfficeDev/office-js#577]
We are unable to fetch the user's `from` address inside their Sent Items folder using the OfficeJs mailbox API.
According to the [documentation](https://docs.microsoft.com/en-us/office/dev/add-ins/reference/objectmodel/requirement-set-1.3/office.context.mailbox.item#from-emailaddressdetails), the `from` email can be read using the following line of code:

```
window.Office.context.mailbox.item.from.emailAddress
```
 
## Expected Behavior
The from email address should be returned by the OfficeJS API.

## Current Behavior
An empty string is returned.

## Steps to Reproduce
Clients tested: 
1. Windows Outlook Desktop 2016, Version: 16.0.4266.1001
2. Windows Outlook Desktop 2013, Version: 15.0.5137.1000
3. Windows Outlook Desktop 2016, Version: 16.0.11629.20246 (Works OK)

Steps: 
- Compose and send a mail
- Navigate to the sent items folder
- Open add-in and execute the following code snippet: 
```
window.Office.context.mailbox.item.from.emailAddress
```

## Context
We have an add-in which can be used to sync emails from Outlook to a third party platform. We fetch the sender and recipients of the mail, it's subject and body, run a few business requirement related operations on it and then sync it to the third party platform. 

## Your Environment

* Platform: PC desktop
* Host: MS Outlook
* Office version number: 16.0.4266.1001, 15.0.5137.1000, and 16.0.11629.20246
* Operating System: Windows
* Browser (if using Office Online): IE 11 (Works)

## Useful logs
Please refer to the screenshot below taken from within the add-in:

<img width="493" alt="Screen Shot 2019-06-26 at 12 53 19 PM" src="https://user-images.githubusercontent.com/40160613/60169026-88ecac00-9823-11e9-9eb5-0e16b3c1b119.png">

**Comment by swasti1994** [comment]
We tested this by adding another email account to Outlook Windows Desktop Client 2016, Version: 16.0.11629.20246, and we observed that the `from` address was an empty string in the sent folder. 
Also, we were able to see the from email in GET message REST API response. So, we wonder if this could be an issue with the Office JS API (and not really dependent on client versions).

**Comment by exextoc** [comment]
This was fixed in Office 2016 Build number 16.0.4556.1000 and later. Your 4266 version needs to be updated to get the fix. It is NOT fixed On Outlook 2013 (15.0.XXXX) builds. Outlook 2013 is no longer being updated. You may need to program a workaround using REST / Get Message if you need this functionality. 

It should be working from 16.0.11629.20246.
